packages/viewer/src/lib/scenes/__tests__/scene.spec.ts

```diff
@@ -294,7 +294,7 @@ describe(Scene, () => {
                       g: 0,
                       r: 0,
                     },
-                    ns: 10,
+                    ns: 50,
```

Comment:
RWS defaults to [4 when all else fails](https://github.com/Vertexvis/rendering-worker-service/blob/6eaa9442088f96d08a2e5c405b8415d82743b42f/src/Embree.cpp#L42).  There's no science behind that number, just something being better than nothing.

Shininess is expressed as an exponent so the effect is asymptotic as the values get larger.

Reply:
Makes sense to me to sync it up with rw.